src/policy/fees.cpp

```diff
@@ -998,19 +998,34 @@ void CBlockPolicyEstimator::FlushUnconfirmed() {
     LogPrint(BCLog::ESTIMATEFEE, "Recorded %u unconfirmed txs from mempool in %gs\n", num_entries, (endclear - startclear)*0.000001);
 }
 
-FeeFilterRounder::FeeFilterRounder(const CFeeRate& minIncrementalFee)
+static std::set<double> MakeFeeSet(const CFeeRate& min_incremental_fee,
+                                   double max_filter_feerate,
+                                   double fee_filter_spacing)
 {
-    CAmount minFeeLimit = std::max(CAmount(1), minIncrementalFee.GetFeePerK() / 2);
+    std::set<double> feeset;
```

Comment:
nit, the second commit would have been easier to review without the localvar renamings, but since you rename them, may as well also update `feeset` to `fee_set` so that they are all updated.

Reply:
Done, renames in a separate commit.